action.yml

```diff
@@ -33,7 +33,7 @@ runs:
         EOF=EOF_OLLAMA_$(openssl rand -hex 12)
         {
           echo "response<<$EOF"
-          ollama run $MODEL "$PROMPT"
+          ollama run $MODEL "${PROMPT@Q}"
```

Comment:
What does the `@Q` do?

Reply:
It escapes characters like double quotes that would break the prompt when included directly.

Reply:
Thanks for the explanation!